dockerfiles/init/modules/openshift/files/scripts/ocp.sh

```diff
@@ -148,15 +151,26 @@ run_ocp() {
 }
 
 deploy_che_to_ocp() {
-    #Repull init image only if IMAGE_PULL_POLICY is set to Always
-    if [ $IMAGE_PULL_POLICY == "Always" ]; then
-        docker pull "$IMAGE_INIT"
+    #Only generate scripts and config files if CHE_GENERATE_SCRIPTS=true
+    if [ $CHE_GENERATE_SCRIPTS == true ]; then
+      echo "OCP generating temporary scripts and configuration files at ${CONFIG_DIR}/instance/config/openshift/scripts/ ."
+      #Repull init image only if IMAGE_PULL_POLICY is set to Always
+      if [ $IMAGE_PULL_POLICY == "Always" ]; then
+          docker pull "$IMAGE_INIT"
+      fi
+      docker run -t --rm -v /var/run/docker.sock:/var/run/docker.sock -v "${CONFIG_DIR}":/data -e IMAGE_INIT="$IMAGE_INIT" -e CHE_MULTIUSER="$CHE_MULTIUSER" eclipse/che-cli:${CHE_IMAGE_TAG} destroy --quiet --skip:pull --skip:nightly
```

Comment:
that is copy paste based on older version of script, please update it accordingly to the current state.
And personally, I don't like the idea of allowing to use local scripts which is error prone.

Reply:
I don't like the idea for people not developing che using but for me it is much easier to use local scripts versus generated. 

Will make the change of the copy paste issue.

Reply:
there is a reason why we generate this, we can merge this but there should be some comments that this is errorprone and may not work as expected

Reply:
Added warning comments.